NetCoreTeamCity/Services/ITestOccurencesService.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.Generic;
+using NetCoreTeamCity.Locators.TestOccurrences;
+using NetCoreTeamCity.Models;
+
+namespace NetCoreTeamCity.Services
+{
+    public interface ITestOccurencesService
+    {
+        TestOccurrence Get(string testOccurenceId);
+        IList<TestOccurrence> Find(TestOccurrencesLocator locator, TestOccurrenceField fields = null);
+        IList<TestOccurrence> Find(TestOccurrencesLocator locator, TestOccurrencesField tosfields = null);
```

Comment:
TestOccurrencesLocator overload wont be needed anymore - it would be good to include Count as optional parameter though